aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/states/AuthorizationState.kt

```diff
@@ -56,11 +60,17 @@ sealed class AuthorizationState : State {
             val resolution = resolveAuthorizationEvent(oldState, event)
             val actions = resolution.actions.toMutableList()
             val builder = Builder(resolution.newState)
+            val deleteUserBuilder = DeleteUserBuilder(resolution.newState)
```

Comment:
deleteUserBuilder not unused (which means it is not required as `DeletingUser` is a substate.

Reply:
removed